mcs/restapi/admin_policies.go

```diff
@@ -211,3 +220,73 @@ func getAddPolicyResponse(params *models.AddPolicyRequest) (*models.Policy, erro
 	}
 	return policy, nil
 }
+
+// policyInfo calls MinIO server to retrieve information of a canned policy.
+// policyInfo() takes a policy name, obtains an []byte (represents a string in JSON format)
+// from the MinIO server and then convert it to *models.Policy , in the future this will change
+// to a Policy struct{} - https://github.com/minio/minio/issues/9171
+func policyInfo(client MinioAdmin, name string) (*models.Policy, error) {
+	policyRaw, err := client.getPolicy(name)
+	if err != nil {
+		return nil, err
+	}
+	var rawPolicy *rawPolicy
+	if err := json.Unmarshal(policyRaw, &rawPolicy); err != nil {
+		return nil, err
+	}
+	policyObject := parseRawPolicy(rawPolicy)
+	policyObject.Name = name
+	return policyObject, nil
+}
+
+// getPolicyInfoResponse performs policyInfo() and serializes it to the handler's output
+func getPolicyInfoResponse(params admin_api.PolicyInfoParams) (*models.Policy, error) {
+	mAdmin, err := newMAdminClient()
+	if err != nil {
+		log.Println("error creating Madmin Client:", err)
+		return nil, err
+	}
+	// create a MinIO Admin Client interface implementation
+	// defining the client to be used
+	adminClient := adminClient{client: mAdmin}
+	policy, err := policyInfo(adminClient, params.Name)
+	if err != nil {
+		log.Println("error getting  group info:", err)
+		return nil, err
+	}
+	return policy, nil
+}
+
+// setPolicy() calls MinIO server to assign policy to a group or user.
+func setPolicy(client MinioAdmin, name, entityName, entityType string) error {
+	isGroup := false
+	if entityType == "group" {
```

Comment:
what are the valid entity types? we can use here similar to what we do on Bucket Access an Enum:
By doing that the server itself will enforce only the valid types when trying to send invalid ones.
```
bucketAccess:
    type: string
    enum:
      - PRIVATE
      - PUBLIC
      - CUSTOM
    default: PRIVATE
  bucket:
    type: object
    required:
      - name
    properties:
      name:
        type: string
        minLength: 3
      size:
        type: integer
        format: int64
      access:
        $ref: "#/definitions/bucketAccess"
      creation_date:
        type: string
```